### PR TITLE
Adopt rich emoji tool

### PR DESCRIPTION
**Pull request by tell-y**
This request is to suggest to adopt rich emoji feature into remote-faces.
I'm not familiar any emoji package so I don't know [emoji mart](https://github.com/missive/emoji-mart) is best one or not.
I'm not sure that broadcasting emoji data as JSON is acceptable.
Thanks.

emoji

Preview URL: https://codesandbox.io/s/github/tell-y/remote-faces/tree/feat/adopt-emoji-mart/web/

### CODE REVIEW

**Comment by dai-shi** [comment]
Looks good. Personally, I want to reduce the bottom margin.

image

File: web/src/hooks/useFaceImages.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ type ImageUrl = string;
 type FaceInfo = {
   nickname: string;
   message: string;
+  emojiJson: string;
```
Comment (dai-shi) [inline web/src/hooks/useFaceImages.ts:10]: We could use JSON directly, which should be more efficient.

Comment (dai-shi) [reply web/src/hooks/useFaceImages.ts:10]: We have a type guard in L23-L27.

**Comment by dai-shi** [comment]
Hmm, to be honest, I think we can just send a native text `emojiJson.native`.

Very nice try anyway. Appreciate for developing this.

Related issue: #46.

**Comment by tell-y** [comment]
Thank you for your review!
I see. I will try your recommended way.
Anyway, I will close this pull request because of conflict and my dirty commits.
I'll send new pull request.

**Pull request closed without merging**